### DISCUSSION TITLE
Escaping {{ in markdown in VitePress

### DISCUSSION THREAD

**Discussion by gerrysweeney** [vuejs/vitepress#480] [Q&A] [answered]
I have the following markdown content

|Name|Type|Description|
|---|---|---|
|value|string|You can use this value in your subject line like this {{value}}

The problem is, no matter what I try, I cannot seem to escape the {{ which Vitepress is interpreting and evaluating the expression within. Not even `{{value}}` works, still gets evaluated

![image](https://user-images.githubusercontent.com/9318317/147865524-9dfada0f-0517-4924-9621-d8ec39e48cdd.png)
This is what I have tried so far, and a few other variations too...

What I anted to get is an output that shows the actual braces and expression like this...

`{{1+1}}`

Any ideas anyone?

**Comment by WebMechanic** [comment]
It's a bit hackish but in the meantime you can add a **thin space** character between the curlies, that's either `&thinsp;` -- if you don't put them in backticks (that would render the HTML entity like here) -- or if you're on Windows you can enter the character code itself: hold down the Alt key and on the **num pad** enter `0157`  -- that's what I use on Twitter to type `@something` w/o triggering a mention. 
I don't know if there's a similar "shortcut" to enter character codes on Mac and Linux. Would love to learn about that.

You can set up your editor to add them as a keyboard macro / snippet / Emmet shortcut.

`{&thinsp;{ foo }}` or `<tt>{&thinsp;{ foo }}</tt>` would become {{ foo }} and `{{ foo }}` respectively. 
Use `<tt>` or `<kbd>` to render the monospaced font w/o using backticks. They're unfortunately deprecated but short to type and still perfectly functional in all browsers, and actually semantic...

**Comment by yyx990803** [comment, marked as answer]
Use this:

```vue
<code v-pre>{{ 1 + 1 }}</code>
```

**Reply by drylint** [reply]
> Use this:
> 
> ```
> <code v-pre>{{ 1 + 1 }}</code>
> ```

Does it make sense? In a regular markdown file, I just need to write \`{{ 1 + 1 }}\`, but migrate to vitepress, I have to write `<code v-pre>{{ 1 + 1 }}</code>`. What if I have a lot of old markdown files having use \`{{ 1 + 1 }}\`. I think it should be same as vuepress, It should not to parse anything (include Vue-style interpolation) in markdown's inline code grammar.

**Reply by brc-dd** [reply]
One can also just wrap complete paragraphs with [`::: v-pre`](https://vitepress.vuejs.org/guide/using-vue.html#escaping)

Alternatively, just do something like this: https://github.com/vuejs/vitepress/discussions/3724#discussioncomment-8963669
